qa/automation/ScoringPortal.Tests.Scenarios/3.Sign_in_with_GoogleAcc/1. sign_in.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 3.1 Sign in with Google Account
+
+Background:
+	Given user turn on VPN with necessary config
+	And go to https://qa.ddso-spot.quantori.com/
+
+Scenario: 3.1_01 User has Google account
+	Given User has a saved Google account
```

Comment:
Which account? In order to check we need to know concrete test data - emails, passwords